tools/vpc-flowlogs-toptalkers/ip-range-labels/main.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import json
+import netaddr
+import urllib.request
+import sys
+import yaml
+
+goog_url="https://www.gstatic.com/ipranges/goog.json"
```

Comment:
please put constants in uppercase, and always add spaces around the `=` sign

Reply:
please run a linter/formatter or manually add spaces around the `=` sign everywhere, except when passing named function args